cli/crates/cli/tests/utils/async_client.rs

```diff
@@ -190,6 +189,31 @@ impl AsyncClient {
             .await
             .unwrap()
     }
+
+    /// Makes a batch GraphQL request.
+    ///
+    /// At the moment this is way less functional than the non-batch request builder
+    /// but is enough to test.
+    pub async fn batch_gql<T>(&self, queries: impl IntoIterator<Item = T>) -> serde_json::Value
+    where
+        T: AsRef<str>,
```

Comment:
Could have been `Into<GqlRequest>` otherwise

Reply:
Oh, I don't think I had that when I was writing this